doc/rst/language/spec/conversions.rst

```diff
@@ -575,14 +573,12 @@ type. Such a conversion does not change the value of the expression.
 Explicit Numeric Conversions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Explicit conversions are allowed from any numeric type or boolean to
+Explicit conversions are allowed from any numeric type or ``bool`` to
 bytes or string, and vice-versa.
 
-When a ``bool`` is converted to a ``bool``, ``int`` or ``uint`` of equal
-or larger size, its value is zero-extended to fit the new
-representation. When a ``bool`` is converted to a smaller ``bool``,
-``int`` or ``uint``, its most significant bits are truncated (as
-appropriate) to fit the new representation.
+When a ``bool`` is converted to an ``int`` or ``uint``, its value is
+zero-extended or truncated to fit the new representation, preserving
+the least-significant bit (which is the only significant one).
```

Comment:
While there, let us replace the unnecessarily-complex story about zero-extension and truncation with something like "A \`\`bool\`\` can be implicitly converted to any integral type by representing \`\`false\`\` as 0 and \`\`true\`\` as 1."